rcl/test/rcl/test_subscription.cpp

```diff
@@ -179,6 +179,12 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
   //                probably using the count_subscriptions busy wait mechanism
   //                until then we will sleep for a short period of time
   std::this_thread::sleep_for(std::chrono::milliseconds(1000));
+#ifdef RMW_TIMESTAMPS_SUPPORTED
+  timespec pre_publish_ts;
+  EXPECT_EQ(0, clock_gettime(CLOCK_REALTIME, &pre_publish_ts)) << " clock_gettime failed";
```

Comment:
This may not be portable, I'd recommend using `std::chrono::system_clock::now()` here instead.

Reply:
Right! I used rcutils now.